src/context/ActionSidebarContext/ActionSidebarContextProvider.tsx

```diff
@@ -62,10 +62,15 @@ const ActionSidebarContextProvider: FC<PropsWithChildren> = ({ children }) => {
   const selectedDomain = useGetSelectedDomainFilter();
   const selectedDomainNativeId = selectedDomain?.nativeId ?? '';
   const navigate = useNavigate();
+  const [searchParams] = useSearchParams();
+  const transactionId = searchParams?.get(TX_SEARCH_PARAM);
```

Comment:
One small comment here: we can rename transactionId to have "param" in its naming for more clarity at 70 line  🙌  But it is not necessary if you don't feel doing it 🫶 

```suggestion
  const transactionIdParam = searchParams?.get(TX_SEARCH_PARAM);
```